doc/lua_api.txt

```diff
@@ -4239,6 +4239,28 @@ Definition tables
         ^ Shall drop item and return the leftover itemstack
         ^ The dropper may be any ObjectRef or nil.
         ^ default: minetest.item_drop ]]
+        on_pickup = func(item_luaentity, up_picker, time_from_last_punch, tool_capabilities, dir),
+        --[[
+        ^ Called when a dropped item is punched and shall be picked up.
+        ^ Shall remove dropped item and add itemstack to `up_picker`'s inventory.
```

Comment:
Why document this if it is already happening, this callback doesn't make this happen.
`up_picker` is bad English, how about `picker` ?

Reply:
How about
`Called when a dropped item is punched and shall be picked up by the 'picker'`
?